src/content/en/2021/css.md

```diff
@@ -107,7 +107,7 @@ Incredibly, this year’s record for the largest number of external stylesheets
   sql_file="stylesheet_count.sql"
 ) }}
 
-Number of style_sheets_ is one thing, but what about the number of actual style _rules_?  Compared to last year, the lower percentiles rose a bit, while the highest barely budged.  What _is_ different in 2021 versus 2020 is that across nearly all percentiles, desktop pages have more rules on average than do mobile pages.
+Number of style sheets is one thing, but what about the number of actual style rules?  Compared to last year, the lower percentiles rose a bit, while the highest barely budged.  What is different in 2021 versus 2020 is that across nearly all percentiles, desktop pages have more rules on average than mobile pages do.
```

Comment:
It's buried in the [Editors' Guide](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Guide) but we do prefer `stylesheets` (one word).

Also I think the emphasis conveyed some of the authors meaning, which might get lost without it. @shantsis do you have a strong opinion about it?

Reply:
Whoops I missed that.
Also I thought the style guide mentioned to avoid overusing italics, but I agree it may lose the meaning here

Reply:
Oh style sheets was 2 words originally, it was just part of the italics. So we can correct that too if we want to keep the italics

Reply:
I would prefer this end up as follows (reduces to one italic, restore ordering at end):

>Number of stylesheets is one thing, but what about the number of actual style rules?  Compared to last year, the lower percentiles rose a bit, while the highest barely budged.  What _is_ different in 2021 versus 2020 is that across nearly all percentiles, desktop pages have more rules on average than do mobile pages.